### PR TITLE
Changed the build type to ament_python and fixed package to run with ros2 run

### PR DESCRIPTION
**Pull request by ahcorde**
Fixed package to run with `ros2 run rqt_plot rqt_plot`. Related to this issue https://github.com/ros-visualization/rqt_console/issues/20

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,12 +1,39 @@
...
+    maintainer_email='dthomas@osrfoundation.org',
+    keywords=['ROS'],
+    classifiers=[
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Programming Language :: Python',
+        'Topic :: Software Development',
+    ],
+    description=(
+        'RQT plugin for monitoring ROS processes.'
```
Comment (dirk-thomas) [inline setup.py]: Copy-n-paste mistake.

**Review by jacobperron — APPROVED** [review]

**Comment by ahcorde** [comment]
Running CI up-to rqt_plot 

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/10908/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6260/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/8883/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/10801/)

**Comment by mabelzhang** [comment]
@ahcorde Should this be merged?

**Comment by jacobperron** [comment]
Since Crystal is EOL, I'd recommend re-targeting this PR to branch for Foxy and/or Rolling.

**Comment by hidmic** [comment]
Rebased on top of `foxy-devel`. I verified it can be `ros2 run` on Windows.

Running CI up to `rqt_msg`, `rqt_srv`, `rqt_service_caller`, `rqt_top`, `rqt_shell`, `rqt_py_console`, `rqt_publisher`, `rqt_plot`:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/14396/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/9170/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/12071/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/14495/)

**Review by ivanpauno — APPROVED** [review]
LGTM!

Same comments than here https://github.com/ros-visualization/rqt_msg/pull/8#pullrequestreview-638063236.

**Pull request merged**